docs/pages/inboxes/push-notifications.md

```diff
@@ -0,0 +1,99 @@
+## Support push notifications
+
+:::warning[🚧 XMTP V3 Alpha]
+
+The XMTP V3 SDKs are in **alpha** status and ready for you to start experimenting with.
+However, we do **not** recommend using alpha software in production apps. Software in this status will change as we add features and iterate based on feedback.
+
+
+XMTP V3 includes breaking changes from V2, most of which will be mitigated by the SDKs. Guidance about the migration path from V2 to V3 will be delivered in forthcoming XMTP V3 release notes.
+
+Join the [XMTP Devs 💪 group chat](https://converse.xyz/group-invite/e-KZyw77m-7sjUmEk5lgu) in Converse to keep up with the latest updates.
+
+:::
+
+With XMTP, you can enable real-time push notifications to keep users updated on new conversations and messages.
+
+For best practices, see [Best practices for push notifications](/notifications/notif-best-practices).
+
+### **Get a Welcome message topic ID**
+
+Get the topic identifier for a new group chat or DM conversation. This topic ID tells your app where to listen on the network for push notifications about the start of a new group chat or DM.
+
+```tsx [React Native]
+// Request
+alix.welcomeTopic()
+
+// Response
+/xmtp/mls/1/w-$installationId/proto
+```
+
+*Replaces V2 `/xmtp/0/invite-$address/proto`* 
+
+### **Get a V3 message topic ID**
+
+Get the topic identifier for a group chat or DM conversation that’s already in progress. This topic ID tells your app where to listen on the network for push notifications about any new messages in a specific group chat or DM conversation.
```

Comment:
this used to say: The conversation of the message

I updated it - but I don’t think I got it quite right?